Chirp/Networking/NetworkService.swift

```diff
@@ -77,5 +84,57 @@ struct NetworkService {
         .eraseToAnyPublisher()
     }
     
+    func createUser(request: CreateUserRequest) -> AnyPublisher<User, Error> {
+        Deferred {
+            Future { promise in
+                db.collection(Table.users.rawValue).document(request.userID).setData([
+                    Key.name.rawValue: request.name,
+                    Key.phoneNumber.rawValue: request.phoneNumber,
+                    Key.profilePicture.rawValue: request.profilePicture ?? ""
+                ]) { error in
+                    if let error = error {
+                        promise(.failure(error))
```

Comment:
After sending error here, you need to return if not it will show error but still call success after this block.
You can either add return or use the if else syntax